providers/ytt/vendir/cni/_ytt_lib/addons/packages/antrea/1.9.0-p1/bundle/config/upstream/interworking.yaml

```diff
@@ -0,0 +1,772 @@
+      - networkpolicies
+      - tiers
+      - clustergroups
+    verbs:
+      - get
+      - watch
+      - list
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - controlplane.antrea.tanzu.vmware.com
+      - controlplane.antrea.io
+    resources:
+      - clustergroupmembers
+      - groupassociations
+    verbs:
+      - get
+      - list
+  - apiGroups:
+      - crd.antrea.tanzu.vmware.com
+    resources:
+      - tierentitlementbindings
+      - tierentitlements
+      - nsxregistrations
+    verbs:
+      - get
+      - watch
+      - list
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - stats.antrea.io
+    resources:
+      - antreaclusternetworkpolicystats
+    verbs:
+      - get
+      - list
+  - apiGroups:
+      - gateway.networking.k8s.io
+    resources:
+      - gateways
+    verbs:
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - config.openshift.io
+    resources:
+      - networks
+    verbs:
+      - get
+      - watch
+      - list
+  - apiGroups:
+      - policy
+    resourceNames:
+      - vmware-system-privileged
+    resources:
+      - podsecuritypolicies
+    verbs:
+      - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app: antrea-interworking
+  name: antrea-interworking
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: antrea-interworking
+subjects:
+  - kind: ServiceAccount
+    name: interworking
+    namespace: vmware-system-antrea
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app: antrea-interworking
+  name: antrea-interworking-supportbundle
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+      - pods/log
+      - nodes
+      - configmaps
+    verbs:
+      - get
+      - list
+  - apiGroups:
+      - "apps"
+    resources:
+      - deployments
+      - replicasets
+      - daemonsets
+    verbs:
+      - list
+  - apiGroups:
+      - system.antrea.io
+    resources:
+      - supportbundles
+    verbs:
+      - get
+      - create
+  - apiGroups:
+      - system.antrea.io
+    resources:
+      - controllerinfos
+      - supportbundles/download
+    verbs:
+      - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app: antrea-interworking
+  name: antrea-interworking-supportbundle
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: antrea-interworking-supportbundle
+subjects:
+  - kind: ServiceAccount
+    name: interworking
+    namespace: vmware-system-antrea
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: register
+  labels:
+    app: antrea-interworking
+    component: register
+  namespace: vmware-system-antrea
+spec:
+  ttlSecondsAfterFinished: 600
+  template:
+    spec:
+      containers:
+        - name: register
+          image: vmware.io/antrea/interworking:0.9.0
```

Comment:
Do we need to use tkg official image repos in this yaml file? cc @hangyan @edwardbadboy

Reply:
I think it should be upstream image ref, it's the same case as antrea upstream yaml